src/components/snew/Markdown/Diff.js

```diff
@@ -45,24 +47,37 @@ const DiffHeader = ({
   </div>
 );
 
-/** TODO: use files diff component when diff implementation is polished  */
-// const FilesDiff = ({ oldFiles, newFiles }) => {
-//   const filesDiff = [];
-//   const hasFile = (file, items) =>
-//     items.filter(f => f.name === file.name && f.payload === file.payload)
-//       .length > 0;
-//   oldFiles.forEach(file => {
-//     if (!hasFile(file, newFiles)) file.removed = true;
-//     filesDiff.push(file);
-//   });
-//   newFiles.forEach(file => {
-//     if (!hasFile(file, filesDiff)) {
-//       file.added = true;
-//       filesDiff.push(file);
-//     }
-//   });
-//   return <ProposalImages files={filesDiff} readOnly={true} />;
-// };
+const DiffBody = ({ body }) => (
+  <div className="md" style={{ padding: "16px" }}>
+    {body}
+  </div>
+);
+
+const getFilesDiff = (newFiles, oldFiles, diffFunc) => [
+  ...newFiles.filter(diffFunc(oldFiles)).map(markAsAdded),
+  ...oldFiles.filter(diffFunc(newFiles)).map(markAsRemoved),
+  ...newFiles.filter(eqFunc(oldFiles)) // for unchanged files
+];
+
+const markAsAdded = elem => ({ ...elem, added: true });
+const markAsRemoved = elem => ({ ...elem, removed: true });
+const diffFunc = arr => elem =>
+  !arr.some(
+    arrelem => arrelem.name === elem.name && arrelem.payload === elem.payload
+  );
+const eqFunc = arr => elem =>
```

Comment:
There's no need to have both `diffFunc` and `eqFunc` logic.

```suggestion
const eqFunc = arr => elem => !diffFunc(arr)(elem);
```